src/components/molecules/MessageList/MessageList.tsx

```diff
@@ -33,30 +32,29 @@ export const MessageList: React.FC<MessageListProps> = ({
   return (
     <>
       <div className="reaction-list small">
-        {usersById &&
-          messages.map((message) => (
-            <React.Fragment
-              key={`${message.from}-${message.to}-${message.ts_utc}`}
-            >
-              {message.from in usersById && (
-                <Message
-                  sender={{ ...usersById[message.from], id: message.from }}
-                  message={message}
-                  onClick={() =>
-                    setSelectedUserProfile({
-                      ...usersById[message.from],
-                      id: message.from, // @debt typing -  User is typed incorrectly so it thinks the id is in usersById
-                    })
-                  }
-                  deletable={allowDelete}
-                  onDelete={() => {
-                    setMessageToDelete(message);
-                    setShowDeleteModal(true);
-                  }}
-                />
-              )}
-            </React.Fragment>
-          ))}
+        {messages.map((message) => (
+          <React.Fragment
+            key={`${message.from}-${message.to}-${message.ts_utc}`}
+          >
+            {message.from in usersById && (
+              <Message
+                sender={{ ...usersById[message.from], id: message.from }}
+                message={message}
+                onClick={() =>
+                  setSelectedUserProfile({
+                    ...usersById[message.from],
+                    id: message.from, // @debt typing -  User is typed incorrectly so it thinks the id is in usersById
+                  })
+                }
+                deletable={allowDelete}
+                onDelete={() => {
+                  setMessageToDelete(message);
```

Comment:
.

Reply:
Have added this to our issue scope! Thank you
https://github.com/sparkletown/internal-sparkle-issues/issues/116